packages/forma-36-react-components/src/components/Button/README.md

```diff
@@ -0,0 +1,117 @@
+
+![Button variations](images/img-1.png)
+
+Contentful buttons are available in 3 different sizes: 
+ - large - 46px height
+ - default - 40px height
+ - small - 30px height
+
+![Button sizes overview](images/img-2.png)
+
+
+## Examples of usage
+
+```jsx
+import {Button} from '@contentful/forma-36-react-components';
+
+<Button buttonType="primary" data-test-id="create-content-type-empty-state">Add content type</Button>
+
+```
+Contentful react button will be renderd as:
+
+```html
+<button data-test-id="create-content-type-empty-state" class="Button__Button___1ZfFj a11y__focus-border--default___60AXp Button__Button--primary___JImeO" type="button">
+  <span tabindex="-1" class="TabFocusTrap__TabFocusTrap___39Vty Button__Button__inner-wrapper___3qrNC">
+   <span class="Button__Button__label___3tcOj">
+    Add content type
+   </span>
+  </span>
+</button>
+```
+
+Buttons  can be used in different contexts in your project. You might want to use it for dropdowns or display icons inside of the button.
+Example of usage of button in the Contentful dropdown:
+
+```jsx
+import {Dropdown, DropdownList, DropdownListItem, Button} from @contentful/forma-36-react-components
+
+<Dropdown
+  onClose={() => setOpenActions(false)}
+  isOpen={openActions}
+  toggleElement={
+     <Button
+      indicateDropdown
+      testId="content-type-actions"
+      onClick={() => {setOpenActions(!openActions)}}
+      buttonType="muted"
+      >
+         Actions
+      </Button>
+  }>
+    <DropdownList>
+    <DropdownListItem>
+       Duplicate
+    </DropdownListItem>
+    <DropdownListItem>
+     Delete
+    </DropdownListItem>
+  </DropdownList>
+</Dropdown>
+```
+
+## Best practices
+
+- Position buttons in consistent places in the interface
+- Reduce complexity by using a small number of actions. Too many actions can create confusion when having to decide
+
+## Content recommendations:
+
+ - UI usage - Button labels should be no longer than 3 words
+ - Webpage usage - Button labels should be no longer than 5 words
+ - Start labels with action verbs. If the button is part of an action dialog, make sure it matches the dialog header.
+ - Use specific words, ideally ones that align with preceding content.
+ - For buttons that are used to cancel destructive actions: label them "Never mind" or similar, instead of cancel. It makes things much easier for users to understand.
+
+## Accessibility
+
+Button component is used to trigger an action or event. By default button is implemented using the native `<button>` element with the proper type="button" provided. If the href element is provided, then the element would be rendered as `<a>` element in the document.
+When a button has focus, both the Space and Enter keys will activate the button. If the button is disabled, then the disabled attribute is provided.
+Contentful buttons are checked in terms of color contrast and pass all the requirements.
+Buttons has trap focus implemented with the tabindex="-1".
```

Comment:
@m10l could you help me out to phrase this focus solution so it would be clear? something like: "users navigating using keyboard, will have the same focus experience"? ayayayayay

Reply:
```suggestion
Buttons have a focus state when using keyboard navigation
```

Not sure it's necessary to give details about the tab focus trap  or the tab index here

Reply:
@m10l I think it might be interesting to explain it. I didn't know it myself and I guess it's a cool info to share with other engineerings.